### PR TITLE
Added labeling instructions

### PR DESCRIPTION
**Pull request by mangomadhava**
Let me know if these look good for anyone wanting to label images.

### CODE REVIEW

File: street2sat_utils/crop_info/instructions_for_labeling.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Instructions for Labeling Images
+
+1. Go to this labeling tool online: https://www.makesense.ai/.
+2. Click get started
+3. Click drop images and select the images you wish to label. Select multiple by holding down the select key.
+4. Click Object Detection
+5. Click Load Labels From File
+6. Upload the classes.txt file
```
Comment (ivanzvonkov) [inline street2sat_utils/crop_info/instructions_for_labeling.md]: So should this file be available in the repository?

**Comment by ivanzvonkov** [comment]
I think a couple of screen shots would not hurt!
I also wonder if it would be easier to just have all labeling information in one place (ie. either here or in the Google Slides presentation).

File: street2sat_utils/crop_info/instructions_for_labeling.md (hunk 1 above)
Comment (mangomadhava) [reply street2sat_utils/crop_info/instructions_for_labeling.md]: Thats a good idea, I'll drop it in the same folder.

**Comment by mangomadhava** [comment]
Added some screenshots

File: street2sat_utils/crop_info/instructions_for_labeling.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+## Instructions for Labeling Images
+
+1. Go to this labeling tool online: https://www.makesense.ai/.
+2. Click 'Get Started'
+3. Click 'Drop Images' and select the images you wish to label. Select multiple by holding down the shift key.
+4. Click Object Detection
+![load_screen](https://github.com/nasaharvest/street2sat_website/blob/labeling_instructions/street2sat_utils/crop_info/assets/opening_screen.png)
+5. Click 'Load Labels' From File
+6. Upload the classes.txt file found here: https://github.com/nasaharvest/street2sat_website/blob/labeling_instructions/street2sat_utils/crop_info/classes.txt
```
Comment (ivanzvonkov) [inline street2sat_utils/crop_info/instructions_for_labeling.md:9]: Might be cleaner to do a relative link here?

**Comment by ivanzvonkov** [comment]
Great, thanks this looks much more interesting with some screenshots :D 
- I suggest using relative links in case the repo name or link ever changes
- Also might be cleaner to put these instructions in the top-level directory, deep folder structures are sometimes intimidating haha

On the whole LGTM though

**Review by ivanzvonkov — APPROVED** [review]

**Pull request merged**